packages/web-app-files/src/composables/viewMode/useViewMode.ts

```diff
@@ -9,7 +9,7 @@ export function useViewMode<T>(options: ComputedRef<string>): ComputedRef<string
 
   const viewModeQuery = useRouteQueryPersisted({
     name: ViewModeConstants.queryName,
-    defaultValue: ViewModeConstants.default
+    defaultValue: ViewModeConstants.default.name
```

Comment:
I've seen this code a bunch of times, maybe worth putting into its own composable